src/CashuMint.ts

```diff
@@ -1,6 +1,9 @@
 import axios from "axios";
-import { Proof } from "./model/Proof.js";
-import { SerealizedBlindedSignature } from "./model/types/SerealizedBlinedSignature.js";
+import {
```

Comment:
Man, sorry it's my fault, but it should be serialized, with i instead of e

Reply:
https://en.m.wikipedia.org/wiki/Serialization

can also use a different prefix altogether, don't know  how fitting it is.